src/server/JasmineGraphServer.cpp

```diff
@@ -1994,31 +1994,53 @@ void JasmineGraphServer::initiateOrgCommunication(std::string graphID, std::stri
 
     Utils utils;
     int fl_clients = stoi(utils.getJasmineGraphProperty("org.jasminegraph.fl_clients"));
-        std::string flagPath = utils.getJasmineGraphProperty("org.jasminegraph.fl.flag.file");
-    int threadLimit = fl_clients + 1;
+    int orgs_count = stoi(utils.getJasmineGraphProperty("org.jasminegraph.fl.num.orgs"));
+    std::string flagPath = utils.getJasmineGraphProperty("org.jasminegraph.fl.flag.file");
+    int threadLimit = fl_clients+1;
```

Comment:
Lets have space in between the `+` symbol.

Reply:
updated in 58678b1a64d640daa8361e0ff3620faf4205adb9

Reply:
Done